DFC.Integration.AVFeed/DFC.Integration.AVFeed.Function.ClearRecycleBin.Console/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("DFC.Integration.AVFeed.Function.ClearRecycleBin.Console")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("HP Inc.")]
```

Comment:
Why HP Inc.?

Reply:
Defaulted to this on my HP laptop! I will change it.